backend/copr_backend/helpers.py

```diff
@@ -634,12 +634,20 @@ def get_chroot_arch(chroot):
 
 def format_evr(epoch, version, release):
     """
-    Return evr in format (epoch:)version-release
+    Return evr in format (epoch:)version-release.  The argument 'epoch' should
+    be integer value or null (but we rather consider "strings" here, too).
     """
-    if epoch is not None and epoch.isdigit():
-        return f"{epoch}:{version}-{release}"
+    if isinstance(epoch, str):
+        if epoch == "":
+            epoch = None
+        elif epoch.isdigit():
+            epoch = int(epoch)
```

Comment:
I think we don't need to convert the epoch to `int` here. The `return f"{epoch}:{version}-{release}"` value will be same in both cases. I think this function can remain as it was before? Maybe only with this small change to work for integers as well?

```diff
-    if epoch is not None and epoch.isdigit():
+    if epoch is not None and (isinstance(epoch, int) or epoch.isdigit()):
         return f"{epoch}:{version}-{release}"
```

Reply:
I had this variant, but it seemed harder to understand to me. But ... :-) I think I like it more now.